charts/egressd/values.yaml

```diff
@@ -102,7 +102,9 @@ collector:
 
   # Explicitly set conntrack mode (netfilter,cilium)
   ctMode: ""
-
+  updateStrategy: RollingUpdate
```

Comment:
This is not common for helm chart. Instead configure default updateStrategy for the whole object

```yaml
updateStrategy:
  type: RollingUpdate
  rollingUpdate:
     maxUnavailable: 1
```

Reply:
Do you mean that the deployment and daemons will share the same update strategy?